ProPublicaCongressAPI/InternalModels/SpecificBill.cs

```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
         [JsonProperty("sponsor")]
         public string SponsorMemberName { get; set; }
 
+        [JsonProperty("sponsor_id")]
+        public string SponsorId { get; set; }
+
         [JsonProperty("sponsor_uri")]
         public string SponsorMemberDetailUrl { get; set; }
 
+        [JsonProperty("sponsor_party")]
+        public string SponsorParty { get; set; }
+
+        [JsonProperty("sponsor_state")]
+        public string SponsorState { get; set; }
+
         [JsonProperty("gpo_pdf_uri")]
         public string BillDocumentPdfUrl { get; set; }
 
+        [JsonProperty("congressdotgov_url")]
+        public string CongressDotGovUrl { get; set; }
+
+        [JsonProperty("govtrack_url")]
+        public string GovTrackUrl { get; set; }
+
         [JsonProperty("introduced_date")]
         public string DateIntroduced { get; set; }
 
+        [JsonProperty("active")]
+        public bool Active { get; set; }
+
+        [JsonProperty("house_passage")]
+        public string HousePassage { get; set; }
+
+        [JsonProperty("senate_passage")]
+        public string SenatePassage { get; set; }
+
+        [JsonProperty("enacted")]
+        public string Enacted { get; set; }
+
+        [JsonProperty("vetoed")]
+        public string Vetoed { get; set; }
+
         [JsonProperty("cosponsors")]
         public int CosponsorCount { get; set; }
 
+        [JsonProperty("withdrawn_cosponsors")]
+        public int WithdrawnCosponsorCount { get; set; }
+
         [JsonProperty("primary_subject")]
         public string PrimarySubject { get; set; }
 
         [JsonProperty("committees")]
         public string Committees { get; set; }
 
+        [JsonProperty("committee_codes")]
+        public string[] CommitteeCodes { get; set; }
+
+        [JsonProperty("subcommittee_codes")]
```

Comment:
I can't find this property in the documentation for the [Specific Bill Endpoint](https://projects.propublica.org/api-docs/congress-api/endpoints/#get-a-specific-bill). Where did you find it?